src/utils/fxa.js

```diff
@@ -155,6 +155,29 @@ async function sendMetricsFlowPing (path) {
 }
 /* c8 ignore stop */
 
+/**
+ * @param {string} subscriptionId 
+ */
+// TODO: Add unit test when changing this code:
+/* c8 ignore start */
+async function deleteSubscription (subscriptionId) {
+  const url = `${AppConstants.OAUTH_ACCOUNT_URI}/oauth/subscriptions/active/${subscriptionId}`
+  try {
+    const response = await fetch(url, {
```

Comment:
Woah, does this really work? Shouldn't we need to pass in a token or something?

Reply:
I get the following when testing it:

```
delete_fxa_subscription {
  stack: 'Internal Server Error: bad response: 401\n'
    '    at deleteSubscription (webpack-internal:///(action-browser)/./src/utils/fxa.js:182:33)\n' +
...
```

Reply:
Redid the logic after testing in heroku. I misunderstood the API. The subscription ID needs to be fetched first.